src/templates/footer.html

```
@@ -15,6 +15,10 @@
       <i title='Report bug' class='fas fa-bug fa-lg'></i>
     </a>
 
+    {% if App.devMode -%}
+      <a title='Interface' aria-label='{{ 'Interface'|trans }}' href='/interface.php'><i title='Interface' class='fa-solid fa-palette fa-fw'></i></a>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)